### ISSUE TITLE
'DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2': undeclared

### ISSUE THREAD

**Issue by mattparks** [glfw/glfw#1320]
After commit https://github.com/glfw/glfw/commit/5294439595e5b95c238a121c8a63c076a0aa2c8a building on `Visual Studio 15 2017 Win64` will result in the following error, resulting and a failed build:
```
"C:\projects\acid\Build\Libraries\glfw\src\glfw.vcxproj" (default target) (12) ->
(ClCompile target) -> 
  c:\projects\acid\libraries\glfw\src\win32_init.c(565): error C2065: 'DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2': undeclared identifier [C:\projects\acid\Build\Libraries\glfw\src\glfw.vcxproj]
```
This is with `BUILD_SHARED_LIBS=OFF` with no other changes made to build.

**Comment by elmindreda** [comment]
Hmm, cannot reproduce with Visual Studio 2017 either with _Open Folder_ or external CMake.

**Comment by mattparks** [comment]
This mainly shows up in a appvenyor project where glfw is a subdirectory: https://ci.appveyor.com/project/Mattparks/acid/build/0.9.8110#L3629
It appers `_DPI_AWARENESS_CONTEXTS_` is defined; however, `DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2` is not defined in this case.

**Comment by elmindreda** [comment]
Cool, then try refreshing from `master` eeac43c3f79cf52c75605e86cb1b1eaf14b01ee6 might help.

**Comment by mattparks** [comment]
I'm still getting errors, https://ci.appveyor.com/project/Mattparks/acid/build/0.9.8111#L3646

**Comment by elmindreda** [comment]
Oh, a very specifically outdated Platform SDK. Will fix.

**Comment by BlackMATov** [comment]
Is there any news? :)

**Comment by elmindreda** [comment]
This may have been fixed by 1725d1c4f5b37c7ccd50dd42a8794b9f6900d12b.
